utils/path.go

```diff
@@ -0,0 +1,30 @@
+package utils
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+)
+
+func FindAbPathInRootfs(path string, rootfs string, sys_paths []string) (string, error) {
```

Comment:
Could you make the variable naming rule like other variables?
like local variables using camel writing ways with first letter lowercase.